### ISSUE TITLE
Contact Us page functionality

### ISSUE THREAD

**Issue by hbh7** [PollBuddy/PollBuddy#275]
**Please describe what has to be done**

- [x] Confirm we want links italicized (aka see if we do that elsewhere), make change if necessary 
- [x] School box should populate from backend call (backend call may not exist, need to validate and possibly create another issue)
- [x] Support form should probably be wider
- [ ] Support form should be submitted to the backend for processing. (backend call may not exist, need to validate and possibly create another issue)
- [x] Some success indicator 
- [ ] Probably don't bother with a loading wheel here for waiting on the school box, just silently do it in the background on page load and show an error in the box until it's loaded ("Loading, please wait", "Error: Unable to load results" or something). In general, unless the backend doesn't work or the user is extremely fast, they shouldn't ever see the message there even though it's there by default.

**Comment by annieshi6** [comment]
support form should be at 50% wide in page-fix

**Comment by fanga2** [comment]
The backend call does not exist. Will have to make a separate issue and probably assign myself to it.
